2023-04-11 15:41:34,105 INFO    StreamThr :22320 [internal.py:wandb_internal():86] W&B internal server running at pid: 22320, started at: 2023-04-11 15:41:34.105342
2023-04-11 15:41:34,106 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: status
2023-04-11 15:41:34,134 INFO    WriterThread:22320 [datastore.py:open_for_write():85] open: E:\YOLOwind\wandb\run-20230411_154134-tf3bxej3\run-tf3bxej3.wandb
2023-04-11 15:41:34,134 DEBUG   SenderThread:22320 [sender.py:send():375] send: header
2023-04-11 15:41:34,136 DEBUG   SenderThread:22320 [sender.py:send():375] send: run
2023-04-11 15:41:34,148 INFO    SenderThread:22320 [sender.py:_maybe_setup_resume():761] checking resume status for None/YOLOv5/tf3bxej3
2023-04-11 15:41:35,105 INFO    SenderThread:22320 [dir_watcher.py:__init__():219] watching files in: E:\YOLOwind\wandb\run-20230411_154134-tf3bxej3\files
2023-04-11 15:41:35,106 INFO    SenderThread:22320 [sender.py:_start_run_threads():1124] run started: tf3bxej3 with start time 1681198894.107335
2023-04-11 15:41:35,122 DEBUG   SenderThread:22320 [sender.py:send_request():402] send_request: summary_record
2023-04-11 15:41:35,122 INFO    SenderThread:22320 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-04-11 15:41:35,129 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: check_version
2023-04-11 15:41:35,129 DEBUG   SenderThread:22320 [sender.py:send_request():402] send_request: check_version
2023-04-11 15:41:35,659 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: run_start
2023-04-11 15:41:35,976 DEBUG   HandlerThread:22320 [system_info.py:__init__():31] System info init
2023-04-11 15:41:35,976 DEBUG   HandlerThread:22320 [system_info.py:__init__():46] System info init done
2023-04-11 15:41:35,976 INFO    HandlerThread:22320 [system_monitor.py:start():181] Starting system monitor
2023-04-11 15:41:35,976 INFO    SystemMonitor:22320 [system_monitor.py:_start():145] Starting system asset monitoring threads
2023-04-11 15:41:35,977 INFO    HandlerThread:22320 [system_monitor.py:probe():201] Collecting system info
2023-04-11 15:41:35,981 INFO    SystemMonitor:22320 [interfaces.py:start():190] Started cpu monitoring
2023-04-11 15:41:35,981 INFO    SystemMonitor:22320 [interfaces.py:start():190] Started disk monitoring
2023-04-11 15:41:35,981 INFO    SystemMonitor:22320 [interfaces.py:start():190] Started gpu monitoring
2023-04-11 15:41:35,988 INFO    SystemMonitor:22320 [interfaces.py:start():190] Started memory monitoring
2023-04-11 15:41:36,001 INFO    SystemMonitor:22320 [interfaces.py:start():190] Started network monitoring
2023-04-11 15:41:36,014 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:36,014 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:36,021 DEBUG   HandlerThread:22320 [system_info.py:probe():195] Probing system
2023-04-11 15:41:36,022 DEBUG   HandlerThread:22320 [git.py:repo():40] git repository is invalid
2023-04-11 15:41:36,022 DEBUG   HandlerThread:22320 [system_info.py:probe():240] Probing system done
2023-04-11 15:41:36,022 DEBUG   HandlerThread:22320 [system_monitor.py:probe():210] {'os': 'Windows-10-10.0.22621-SP0', 'python': '3.9.16', 'heartbeatAt': '2023-04-11T07:41:36.021310', 'startedAt': '2023-04-11T07:41:34.091411', 'docker': None, 'cuda': None, 'args': ('--cfg', 'models/yolov5s.yaml', '--data', 'wind-data/wind.yaml', '--hyp', 'E:/YOLOwind/data/hyps/RM.wind.yaml', '--epochs', '500', '--batch-size', '8', '--img-size', '640', '--noval', '--adam', '--workers', '1', '--negative-path', 'D:/300-600/images/train'), 'state': 'running', 'program': 'E:\\YOLOwind\\train.py', 'codePath': 'train.py', 'host': 'DESKTOP-LP6CIGL', 'username': 'Lenovo', 'executable': 'E:\\anaconda\\envs\\YOLOwind\\python.exe', 'cpu_count': 4, 'cpu_count_logical': 8, 'cpu_freq': {'current': 1609.0, 'min': 0.0, 'max': 2112.0}, 'cpu_freq_per_core': [{'current': 1609.0, 'min': 0.0, 'max': 2112.0}], 'disk': {'total': 179.8857421875, 'used': 155.23410415649414}, 'gpu': 'NVIDIA GeForce MX350', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce MX350', 'memory_total': 2147483648}], 'memory': {'total': 15.834033966064453}}
2023-04-11 15:41:36,022 INFO    HandlerThread:22320 [system_monitor.py:probe():211] Finished collecting system info
2023-04-11 15:41:36,022 INFO    HandlerThread:22320 [system_monitor.py:probe():214] Publishing system info
2023-04-11 15:41:36,022 DEBUG   HandlerThread:22320 [system_info.py:_save_pip():51] Saving list of pip packages installed into the current environment
2023-04-11 15:41:36,022 DEBUG   HandlerThread:22320 [system_info.py:_save_pip():67] Saving pip packages done
2023-04-11 15:41:36,023 DEBUG   HandlerThread:22320 [system_info.py:_save_conda():74] Saving list of conda packages installed into the current environment
2023-04-11 15:41:36,108 INFO    Thread-16 :22320 [dir_watcher.py:_on_file_created():278] file/dir created: E:\YOLOwind\wandb\run-20230411_154134-tf3bxej3\files\requirements.txt
2023-04-11 15:41:36,109 INFO    Thread-16 :22320 [dir_watcher.py:_on_file_created():278] file/dir created: E:\YOLOwind\wandb\run-20230411_154134-tf3bxej3\files\wandb-summary.json
2023-04-11 15:41:36,109 INFO    Thread-16 :22320 [dir_watcher.py:_on_file_created():278] file/dir created: E:\YOLOwind\wandb\run-20230411_154134-tf3bxej3\files\conda-environment.yaml
2023-04-11 15:41:38,249 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:38,249 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:40,486 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:40,487 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:42,722 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:42,723 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:43,168 DEBUG   HandlerThread:22320 [system_info.py:_save_conda():86] Saving conda packages done
2023-04-11 15:41:43,168 INFO    HandlerThread:22320 [system_monitor.py:probe():216] Finished publishing system info
2023-04-11 15:41:43,182 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: status_report
2023-04-11 15:41:43,182 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: keepalive
2023-04-11 15:41:43,183 DEBUG   SenderThread:22320 [sender.py:send():375] send: files
2023-04-11 15:41:43,183 INFO    SenderThread:22320 [sender.py:_save_file():1378] saving file wandb-metadata.json with policy now
2023-04-11 15:41:43,196 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: stop_status
2023-04-11 15:41:43,196 DEBUG   SenderThread:22320 [sender.py:send_request():402] send_request: stop_status
2023-04-11 15:41:43,671 DEBUG   SenderThread:22320 [sender.py:send():375] send: telemetry
2023-04-11 15:41:43,672 DEBUG   SenderThread:22320 [sender.py:send():375] send: config
2023-04-11 15:41:44,139 INFO    Thread-16 :22320 [dir_watcher.py:_on_file_modified():295] file/dir modified: E:\YOLOwind\wandb\run-20230411_154134-tf3bxej3\files\conda-environment.yaml
2023-04-11 15:41:44,140 INFO    Thread-16 :22320 [dir_watcher.py:_on_file_created():278] file/dir created: E:\YOLOwind\wandb\run-20230411_154134-tf3bxej3\files\output.log
2023-04-11 15:41:44,140 INFO    Thread-16 :22320 [dir_watcher.py:_on_file_created():278] file/dir created: E:\YOLOwind\wandb\run-20230411_154134-tf3bxej3\files\wandb-metadata.json
2023-04-11 15:41:44,153 INFO    wandb-upload_0:22320 [upload_job.py:push():137] Uploaded file C:\Users\Lenovo\AppData\Local\Temp\tmpjl8pwzuawandb\y2ycpsc3-wandb-metadata.json
2023-04-11 15:41:44,737 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:44,738 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:45,139 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: status_report
2023-04-11 15:41:46,759 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:46,760 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:48,818 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:48,819 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:50,676 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: status_report
2023-04-11 15:41:50,913 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:50,914 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:52,992 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:52,993 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:55,270 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:55,272 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:55,628 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: status_report
2023-04-11 15:41:57,723 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:57,724 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:58,193 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: stop_status
2023-04-11 15:41:58,194 DEBUG   SenderThread:22320 [sender.py:send_request():402] send_request: stop_status
2023-04-11 15:41:59,781 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:41:59,782 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:01,694 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: status_report
2023-04-11 15:42:01,835 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:01,836 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:03,856 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:03,856 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:05,874 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:05,876 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:06,759 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: status_report
2023-04-11 15:42:07,916 INFO    Thread-16 :22320 [dir_watcher.py:_on_file_modified():295] file/dir modified: E:\YOLOwind\wandb\run-20230411_154134-tf3bxej3\files\config.yaml
2023-04-11 15:42:08,182 ERROR   gpu       :22320 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
2023-04-11 15:42:08,195 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:08,196 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:10,218 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:10,218 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:12,079 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: status_report
2023-04-11 15:42:12,236 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:12,237 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:13,204 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: stop_status
2023-04-11 15:42:13,206 DEBUG   SenderThread:22320 [sender.py:send_request():402] send_request: stop_status
2023-04-11 15:42:14,259 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:14,261 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:16,272 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:16,272 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:17,149 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: status_report
2023-04-11 15:42:18,529 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:18,529 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:20,539 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:20,539 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:22,156 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: status_report
2023-04-11 15:42:22,547 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:22,548 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:24,557 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:24,559 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:26,567 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:26,567 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:28,056 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: status_report
2023-04-11 15:42:28,230 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: stop_status
2023-04-11 15:42:28,254 DEBUG   SenderThread:22320 [sender.py:send_request():402] send_request: stop_status
2023-04-11 15:42:28,912 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:28,929 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:31,075 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:31,088 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:33,099 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:33,099 ERROR   gpu       :22320 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2023-04-11 15:42:33,648 DEBUG   HandlerThread:22320 [handler.py:handle_request():144] handle_request: status_report
2023-04-11 15:42:34,539 INFO    cpu       :22320 [interfaces.py:monitor():140] Process cpu has exited.
2023-04-11 15:42:34,540 DEBUG   SystemMonitor:22320 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2023-04-11 15:42:34,540 DEBUG   SystemMonitor:22320 [system_monitor.py:_start():166] Finished system metrics aggregation loop
2023-04-11 15:42:34,542 DEBUG   SystemMonitor:22320 [system_monitor.py:_start():170] Publishing last batch of metrics
2023-04-11 15:42:34,544 DEBUG   SenderThread:22320 [sender.py:send():375] send: stats
2023-04-11 15:42:34,841 ERROR   gpu       :22320 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
2023-04-11 15:42:35,235 INFO    MainThread:22320 [internal.py:handle_exit():76] Internal process exited
